ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py

```diff
@@ -349,26 +348,14 @@ def _makeEvent(evt):
 
         for evt in event_results:
             data['events'].append(_makeEvent(evt))
-
         data['events'].append({
             'device': config.id,
-            'eventClass': '/Status',
             'summary': 'Windows EventLog: successful event collection',
             'severity': ZenEventClasses.Clear,
             'eventKey': 'WindowsEventCollection: {}'.format(ds0.params.get('eventid', '')),
-            'eventClassKey': 'WindowsEventLogSuccess',
+            'eventClassKey': 'WindowsEventLogCollection',
         })
 
-        if 'ps_err_msg' not in locals():
```

Comment:
this event still needs to be sent to clear any previous powershell errors.  also, change eventClassKey to WindowsEventLogCollection and remove the eventClass